.stylelintrc.json

```diff
@@ -16,20 +16,25 @@
     "function-name-case": "lower",
     "function-parentheses-newline-inside": "always-multi-line",
     "function-parentheses-space-inside": "never",
-    "function-url-data-uris": "never",
+    "function-url-data-uris": null,
     "function-url-quotes": "always",
     "function-whitespace-after": "always",
-    "number-leading-zero": "never",
+    "number-leading-zero": "always",
```

Comment:
Having the zero helps to avoid mistakes when someone doesn't notice the point, `.1` vs `0.1`. Also helps to convey that the author truly meant `0.1` and `.1` isn't a typo.